packages/paste-website/src/components/site-wrapper/sidebar/SidebarNestedItem.tsx

```diff
@@ -3,7 +3,13 @@ import {Box} from '@twilio-paste/box';
 
 export const SidebarNestedItem: React.FC = props => {
   return (
-    <Box as="li" marginTop="space20" marginBottom="space20" fontSize="fontSize30" lineHeight="lineHeight20">
+    <Box
+      as="li"
+      marginTop="space20"
+      marginBottom="space20"
+      fontSize={['fontSize50', 'fontSize50', 'fontSize30']}
```

Comment:
Just want to verify the fontSize should get larger on tablet and mobile sizes.

Reply:
According to the designs, yup